static/src/javascripts/projects/common/modules/commercial/donot-use-adblock.js

```diff
@@ -20,23 +22,42 @@ define([
     svgs
 ) {
     function init() {
-        var alreadyVisted = storage.local.get('alreadyVisited') || 0,
-            adblockLink = 'https://membership.theguardian.com?INTCMP=adb-mv';
+        var alreadyVisted = storage.local.get('alreadyVisited') || 0;
```

Comment:
No problem, perhaps we should add this to the [eslint config](http://eslint.org/docs/rules/one-var.html) then? /cc @OliverJAsh